docs/2.0/docs/pipelines/installation/viamachineusers.md

```diff
@@ -369,3 +383,64 @@ For more information on creating and using GitHub Actions Repository secrets, re
 
 </TabItem>
 </Tabs>
+
+</TabItem>
+<TabItem value="gitlab" label="GitLab">
```

Comment:
General comment/observation: when using tabs, the top level headers displayed on the right TOC does not change.

In this section, it leads to potential confusion because I am seeing headers relevant only to GitHub. This may be one  case, for a later ticket, where separating large content into separate pages works better than a tab.

Reply:
The whole "viamachineusers" page needs... more TLC.  I'll file a ticket!